tests/IResearch/IResearchQuerySelectAllTest.cpp

```diff
@@ -36,7 +36,6 @@
 #include "VocBase/LogicalCollection.h"
 #include "store/mmap_directory.hpp"
 #include "utils/index_utils.hpp"
-#include "utils/string_utils.hpp"
```

Comment:
absl::Substitute is now used by this test. Should the corresponding header file be included here?